### DISCUSSION TITLE
Index WikiJS in search engines

### DISCUSSION THREAD

**Discussion by obed-vazquez** [requarks/wiki#4782] [Help / Questions] [answered]
Is it possible for us to index our entire Wiki in a search engine?
I have a hosting and a domain, but I'm not sure of how, or if we even can add our wiki(s) to any search engine for its contents to be discovered/found in a Google search for example.

**Comment by NGPixel** [comment, marked as answer]
As long as your pages are reachable from other pages, Googlebot will index them just fine.
